crates/aptos/src/move_tool/mod.rs

```diff
@@ -831,6 +831,10 @@ impl CliCommand<TransactionSummary> for PublishPackage {
         "PublishPackage"
     }
 
+    fn jsonify_error_output(&self) -> bool {
+        false
+    }
```

Comment:
What's this property?  I don't see it being used, and it's always false?

Reply:
The problem is that the cli will jsonfy the output and it looks pretty ugly if the string has new line.

Reply:
Yeah, but I don't see how this has anything to deal with the original pull request.  And if anyone's parsing the JSON output, it now will just disappear.  This happens with things like the Move playground

Reply:
I can revert the changes here. It’s simply for atheistic purposes for cli